server/src/main/java/org/opensearch/gateway/remote/RemoteClusterStateService.java

```diff
@@ -733,6 +733,24 @@ private void writeMetadataManifest(String clusterName, String clusterUUID, Clust
         );
     }
 
+    ThreadPool getThreadpool() {
+        return threadpool;
+    }
+
+    BlobStore getBlobStore() {
+        return blobStoreRepository.blobStore();
+    }
+
+    private String fetchPreviousClusterUUID(String clusterName, String clusterUUID) {
```

Comment:
Where is this method used?

Reply:
This was present from before, but it was unused and removed in https://github.com/opensearch-project/OpenSearch/pull/13150/files#r1567193608, missed this during merge from main. Removing from my changes as well.